articles/azure-maps/choose-map-style.md

```diff
@@ -85,6 +83,74 @@ map.setStyle({ style: 'satellite' });
 > [!VIDEO //codepen.io/azuremaps/embed/yqXYzY/?height=265&theme-id=0&default-tab=js,result&embed-version=2&editable=true]
 ----------------------------------------------------------------------------------------------->
 
+## Customize the base map
+The base map can be customized with the `styleOverrides` option. To see all available options, see [styleOverrides].
+
+The following code shows how to disable country/region borders and building footprints on the map.
+
+```javascript
+var map = new atlas.Map('map', {
+    styleOverrides: {
+        countryRegion: { borderVisible: false },
+        buildingFootprint: { visible: false },
+    },
+
+    //Additional map options
+});
+```
+
+Here are the effects of all five available `styleOverrides` options:
+
+1. Country/region borders
+   
+```javascript
+styleOverrides: {
+    countryRegion: { borderVisible: false }
+}
+```
+:::image type="content" source="./media/choose-map-style/country-region.gif" alt-text="Country/region borders":::
+
+2. Admin district borders
+
+```javascript
+styleOverrides: {
+    adminDistrict: { borderVisible: false }
+}
+```
+:::image type="content" source="./media/choose-map-style/admin-district.gif" alt-text="Admin district borders":::
+
+3. Second admin district borders
+
+```javascript
+styleOverrides: {
+    adminDistrict2: { borderVisible: false }
+}
+```
+:::image type="content" source="./media/choose-map-style/admin-district2.gif" alt-text="Second admin district borders":::
+
+4. Building footprints
+
+```javascript
+styleOverrides: {
+    buildingFootprint: { visible: false }
+}
+```
+:::image type="content" source="./media/choose-map-style/building-footprint.gif" alt-text="Building footprints":::
+
+5. Road details
+
+```javascript
+styleOverrides: {
+    roadDetails: { visible: false }
+}
+```
+:::image type="content" source="./media/choose-map-style/road-details.gif" alt-text="Road details":::
+
+> [!NOTE]
+> For the supported map styles on each `styleOverrides` option, please read [styleOverrides] API documentation for more details.
+
+For a fully functional sample that shows how the different styles affect how the map is rendered, see [Map style options] in the [Azure Maps Samples]. For the source code for this sample, see [Map style options source code].
```

Comment:
For a sample showing the different styles and how they affect how the map is rendered, see [Map style options] in the [Azure Maps Samples]. For this samples source code, see [Map style options source code].